app/assets/stylesheets/resource_curriculum_show.scss

```diff
@@ -1,42 +1,45 @@
 @import 'color';
 
-
-.resources_container a, .curriculums_container a {
-  color: $brick;
-  &:hover {
-    color: lighten( $brick, 10% );
-    text-decoration: underline;
+.resources_container, .curriculums_container {
+  a {
+    color: $brick;
+    &:hover {
+      color: lighten( $brick, 10% );
```

Comment:
Expected 0 spaces after `(` instead of ` `<br>Expected 0 spaces before `)` instead of ` `